src/components/TourPoint/TourPoint.types.ts

```diff
@@ -30,6 +30,13 @@ export interface Props {
   step: number;
   style?: CSSProperties;
   title?: HTMLElement['title'];
+  getStepsTranslation?: ({
+    step,
+    steps,
+  }: {
+    step: number;
+    steps: number;
```

Comment:
nit: Could this be `currentStep` and `totalSteps` to be more clear.

Reply:
done